go/ai/gen.go

```diff
@@ -27,16 +82,17 @@ type GenerateActionOptions struct {
 	MaxTurns           int                          `json:"maxTurns,omitempty"`
 	Messages           []*Message                   `json:"messages,omitempty"`
 	Model              string                       `json:"model,omitempty"`
-	Output             *GenerateActionOptionsOutput `json:"output,omitempty"`
+	Output             *GenerateActionOutputConfig  `json:"output,omitempty"`
+	Resume             *GenerateActionOptionsResume `json:"resume,omitempty"`
 	ReturnToolRequests bool                         `json:"returnToolRequests,omitempty"`
 	ToolChoice         ToolChoice                   `json:"toolChoice,omitempty"`
-	Tools              []string                     `json:"tools,omitempty"`
+	Tools              []*ToolDefinition            `json:"tools,omitempty"`
```

Comment:
This is still `[]string`: https://github.com/firebase/genkit/blob/103b7716f5f6004b46cce2129c4fe6d6e7bdb804/genkit-tools/common/src/types/model.ts#L359